src/ci/docker/scripts/android-start-emulator.sh

```diff
@@ -11,6 +11,6 @@ export SHELL=/bin/bash
 # classic engine the emulator starts with the current date and the tests run
 # fine. If another image is used, this need to be evaluated again.
 nohup nohup emulator @armeabi-v7a-18 \
-    -engine classic -no-window -partition-size 2047 0<&- 2>&1 /dev/null &
+    -engine classic -no-window -partition-size 2047 0<&- 2>&1 > /dev/null &
```

Comment:
This is not the same behavior: `2>&1` redirects to where stdout points _currently_, which is the terminal. You need to redirect _after_ you change stdout to get the previous behavior.
```suggestion
    -engine classic -no-window -partition-size 2047 0<&- >/dev/null 2>&1 &
```